src/main/java/me/snowdrop/fallback/Fallback.java

```diff
@@ -37,4 +35,27 @@
      * This method can either have zero parameters, or a single param of type me.snowdrop.fallback.ExecutionContext
      */
     String fallbackMethod() default  "error";
+
+    /**
+     * The type of exception that this Fallback method handles
+     *
+     * By default all exceptions are handled
+     *
+     * This is particularly useful when multiple fallbacks are configured for the same method
+     */
+    Class<? extends Exception> exception() default Exception.class;
+
+    /**
+     * Order in which to the fallback will be used
+     * smaller values mean higher order
```

Comment:
I am alwasy confused with what exactly means higher, lower, but I know it is something that is on my brain. Maybe I will add smaller values mean higher order so are executed first/last